pkg/api/handlers/types.go

```diff
@@ -347,24 +347,35 @@ func ImageDataToImageInspect(ctx context.Context, l *libpodImage.Image) (*ImageI
 
 }
 
-func LibpodToContainer(l *libpod.Container, infoData []define.InfoData) (*Container, error) {
+func LibpodToContainer(l *libpod.Container, infoData []define.InfoData, sz bool) (*Container, error) {
 	imageId, imageName := l.Image()
 
 	var (
 		err        error
 		sizeRootFs int64
 		sizeRW     int64
 		state      define.ContainerStatus
+		stateStr   string
 	)
 
 	if state, err = l.State(); err != nil {
 		return nil, err
 	}
-	if sizeRW, err = l.RWSize(); err != nil {
-		return nil, err
+	stateStr = state.String()
+	if stateStr == "configured" {
+		stateStr = "created"
 	}
-	if sizeRootFs, err = l.RootFsSize(); err != nil {
-		return nil, err
+
+	if sz {
+		if sizeRW, err = l.RWSize(); err != nil {
+			return nil, err
+		}
+		if sizeRootFs, err = l.RootFsSize(); err != nil {
+			return nil, err
+		}
+	} else {
+		sizeRW = 0
```

Comment:
These are defaults so no reason for this code block.

Reply:
Done